chcemvediet/apps/inforequests/templates/inforequests/index/index.html

```diff
@@ -16,5 +16,9 @@
   <h1>{% trans 'inforequests:index:heading' %}</h1>
   {% include "inforequests/index/texts/index.html" %}
   {% include "inforequests/index/published.html" with published_inforequests=inforequest_page.object_list %}
-  {% include "main/snippets/paging.html" with page=inforequest_page %}
+  {% if inforequest_page %}
+    {% include "main/snippets/paging.html" with page=inforequest_page %}
```

Comment:
Nebolo by logickejsie dat `  {% if inforequest_page %}` pred `{% include "inforequests/index/published.html" ... %}`?

Reply:
Uvažoval som nad tým, samotný `inforequests/index/published.html` templte má na začiatku v sebe podmienku `{% if published_inforequests %}`. Preto som to dal až za. :ok:

Reply:
Aha, to som si nevsimol. Kazdopadne bolo divne, ak sa zaroven importovalo aj `published.html` aj `published_inforequests_empty.html`. Takto je to teraz imho lepsie.